test/it/IleQueryDisabled.scala

```diff
@@ -0,0 +1,6 @@
+import play.api.Configuration
+trait IleQueryDisabled extends IntegrationSpec {
```

Comment:
IMHO this is a code smell. I understand that this trait is simply not to repeat yourself.
But in every Spec both `IleQueryDisabled` and `IntegrationSpec` are mixed-in.
I'd rather override the method in every Spec it is needed.

Reply:
I started doing that and after the 3rd or 4th time I thought, this would look better with a "with.... " trait.    

There's no pleasing some people.

Reply:
I see. But then it makes no point in mixing both in concrete Specs.